Godeps/_workspace/src/github.com/miekg/dns/types.go

```diff
@@ -1567,9 +1567,10 @@ type CAA struct {
 
 func (rr *CAA) Header() *RR_Header { return &rr.Hdr }
 func (rr *CAA) copy() RR           { return &CAA{*rr.Hdr.copyHeader(), rr.Flag, rr.Tag, rr.Value} }
-func (rr *CAA) len() int           { return rr.Hdr.len() + 1 + len(rr.Tag) + len(rr.Value)/2 }
-func (rr *CAA) String() string     { return rr.Hdr.String() + strconv.Itoa(int(rr.Flag)) + " " + rr.Tag + " " + sprintCAAValue(rr.Value) }
```

Comment:
I didn't see this mentioned in your PR description so: Were the changes to Godeps edits you made or did you run a godeps save here? If the latter, could we do this in another PR (perhaps one that this one depends on) or, at least, mention what was needed? If the former, let's fork the thing into our repo and upstream the patch.